src/SpectatorView.Unity/Assets/SpectatorView/Scripts/Sharing/SpatialCoordinateSystemManager.cs

```diff
@@ -42,7 +42,9 @@ public class SpatialCoordinateSystemManager : Singleton<SpatialCoordinateSystemM
 
         internal const string CoordinateStateMessageHeader = "COORDSTATE";
         private const string LocalizeCommand = "LOCALIZE";
+        private const string LocalizationCompleteCommand = "LOCALIZE_Complete";

```

Comment:
>LOCALIZE_Complete [](start = 60, length = 17)

nit: we have been using all cap names here so this one looks a bit strange. Maybe LOCALIZECOMP?

Reply:
Sure thing, named it LOCALIZEDONE

---
In reply to: [295545183](https://github.com/microsoft/MixedReality-SpectatorView/pull/26#discussion_r295545183) [](ancestors = 295545183)